### PR TITLE
Prometheus: fix parsing of infinite sample values (#28287)

### LINKED ISSUE #28287: Prometheus: infinite sample values are rendered as NaN
**What happened**:
`+Inf` sample value returned from prometheus is rendered as `NaN` in table view.

**What you expected to happen**:
`+Inf` sample value returned from prometheus to be rendered as `∞` in table view.

**How to reproduce it (as minimally and precisely as possible)**:
image
Visualization: Table

**Environment**:
- Grafana version: 6.7.3
- Data source type & version: Prometheus, 2.17.2
- OS Grafana is installed on: Linux
- User OS & Browser: Linux, Chrome
- Grafana plugins:
- Others:

### PR DESCRIPTION
**Pull request by KonanMentor**
**What this PR does / why we need it**:
Correctly parse `+Inf`, `-Inf` prometheus sample values.

**Which issue(s) this PR fixes**: #28287

Fixes #28287

### CODE REVIEW

File: public/app/plugins/datasource/prometheus/result_transformer.ts [hunk 1]
```diff
@@ -313,3 +316,14 @@ function parseHistogramLabel(le: string): number {
   }
   return Number(le);
 }
+
+function parseSampleValue(value: string): number {
```
Comment (zoltanbedi) [inline public/app/plugins/datasource/prometheus/result_transformer.ts:320]: Thanks for your contribution @KonanMentor this looks good. Can you please use your function in places where parseHistogramLabel used as well?

**Review by zoltanbedi — CHANGES_REQUESTED** [review]

File: public/app/plugins/datasource/prometheus/result_transformer.ts (hunk 1 above)
Comment (KonanMentor) [reply public/app/plugins/datasource/prometheus/result_transformer.ts:313]: Done, please check

**Review by zoltanbedi — APPROVED** [review]
Good job! Thank you!

**Review by ivanahuckova — APPROVED** [review]
LGTM! This is awesome! Thanks Aliaksei for your contribution.🙂

**Comment by kylebrandt** [comment]
Removed cp label because it is in beta2, and see this in the v7.3.x branch already

**Pull request merged**